package.json

```diff
@@ -25,6 +23,7 @@
         "test:ci": "vitest run",
         "changeset": "changeset",
         "changeset:release": "pnpm build && changeset publish",
+        "anvil": "anvil --fork-url $(. ./.env && echo $ETHEREUM_RPC_URL)",
```

Comment:
This is only needed until Relayer V6 is deployed right? Note to remove once we have updated tests.

Reply:
@brunoguerios do we have an expected date for this?

Reply:
Not yet. It's currently being audited and after that it needs to go through governance approval. My guess is that it will be live in ~1 month.